### DISCUSSION TITLE
Legal Question about Github Pages

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22800] [Other Feature Feedback, Questions, & Ideas]
<p>Can you use a license DIFFERENT than the ones on <a href="http://choosealicense.com" rel="noopener nofollow ugc">choosealicense.com</a>?</p>
<p>Like, the only options are CC BY and CC BY-SA, but i want CC BY-NC-ND</p>
<p>Any ideas?</p>

**Comment by sr229** [comment, marked as answer]
<p>You can use any kind of license beyond what was listed in <a href="http://choosealicense.com" rel="noopener nofollow ugc">choosealicense.com</a>. GitHub will not remove your repository if you are the legal owner of the repository.</p>
